app/templates/counselor/calendar.html

```diff
@@ -0,0 +1,6 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+    <iframe src="https://calendar.google.com/calendar/embed?src=ngshack4impact%40gmail.com&ctz=America/New_York" style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
```

Comment:
Can we make the calendar link a variable in `config.py` instead of a static string? This will allow our client in the future to change the link as needed without searching through our code!